### ISSUE TITLE
Question about one of your examples

### ISSUE THREAD

**Issue by ldidry** [gryphonshafer/Email-Mailer#5]
I have a question about the example [line 226](https://github.com/gryphonshafer/Email-Mailer/blob/master/#L226) of `lib/Email/Mailer.pm`.
```perl
# setup a second mail object based on the first but changing the "from"
my $mail_0 = Email::Mailer->new(
    from    => $from,
    subject => $subject,
    html    => $html,
);
my $mail_1->new( from => 'different_address@example.com' );
$mail_0->send;
$mail_1->send;
```

Should the be the `$mail_1`’s construction be like that?
```perl
my $mail_1 = $mail_0->new( from => 'different_address@example.com' );
```

**Comment by gryphonshafer** [comment]
Ah, yes. Good point. Looks like a POD bug. Thanks for reporting it.

**Comment by gryphonshafer** [comment]
Resolved with 37b9503.
